nginx_snippets/http/01-submatrix_upstream.conf

```diff
@@ -0,0 +1,3 @@
+upstream submatrix {
+	server unix:/run/nginx/socks/submatrix.sock;
+}
```

Comment:
is this upstream necessary? can we not just do `proxy_pass http://unix:/run/nginx/socks/submatrix.sock;`